packages/tpetra/core/example/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ ADD_SUBDIRECTORY(Lesson06-Custom-Operator)
 ADD_SUBDIRECTORY(Lesson07-Kokkos-Fill)
 
 ADD_SUBDIRECTORY(Finite-Element-Assembly)
+ADD_SUBDIRECTORY(BlockCrs)
```

Comment:
Should we put benchmarks here, or move tpetra/core/example/advanced/Benchmarks up a directory and put all the benchmarks in there?